src/components/Utils/resources.js

```diff
@@ -251,3 +253,16 @@ export const ORDERS = {
   path: ORDERS_API,
   records: 'purchaseOrders',
 };
+
+export const AGREEMENT_LINES = {
+  ...baseManifest,
+  accumulate: true,
+  fetch: false,
+  limitParam: 'perPage',
+  path: AGREEMENT_LINES_API,
+  GET: {
+    params: {
+      filters: 'poLines.poLineId==!{lineId}',
```

Comment:
`mod-agreements` module is written on Groovy and doesn't support `params: { query }` - it returns all items with any `query`

Reply:
do you need params config in this case?

Reply:
or do they support filters instead of query?

Reply:
yes. With `filters` it returns requested data

Reply:
`lineId` is a magic string. Basically it's a coupling to param name in location without any reason, usually we do such things in .GET({params}), so we can put required param to the deps list. 
also it's a good idea to check the limit bug

Reply:
fixed, for controlling limits it requires `perPage`